popde/adaptive_kde.py

```diff
@@ -2,6 +2,8 @@
 import scipy
 from density_estimate import VariableBwKDEPy 
 from scipy.stats import gmean
+from sklearn.model_selection import KFold, LeaveOneOut
```

Comment:
it is probably better to import these only when needed, ie inside the relevant methods, since it would be good to be able to run the code for other classes (ie if using a class without grid optimization) without needing to install and import sklearn

Reply:
fixed and called in relevant method